src/plugins/simpleini/README.md

```diff
@@ -11,7 +11,9 @@
 
 This plugin reads and writes files written in a basic line-oriented ini-like format.
 It is very simplistic without sections, the [ini](../ini/) plugin and for specifications
-the [ni](../ni/) plugin should be preferred in most cases.
+the [ni](../ni/) plugin should be preferred in most cases. Since the plugin requires
+the GNU C library it **will not work** on operating systems that use another C library
```

Comment:
Are you sure? Afaik this statement (that it wont work in macOS) is only true for `simpleini`, not for `ni`.

Reply:
> Afaik this statement (that it wont work in macOS) is only true for `simpleini`, not for `ni`.

That is exactly what I meant 😊. I replaced “this plugin“ with “`simpleini` plugin“. Hope that works for you.